pyairtable/api/abstract.py

```diff
@@ -16,25 +16,31 @@
 
 class ApiAbstract(metaclass=abc.ABCMeta):
     VERSION = "v0"
-    API_BASE_URL = "https://api.airtable.com/"
     API_LIMIT = 1.0 / 5  # 5 per second
-    API_URL = posixpath.join(API_BASE_URL, VERSION)
     MAX_RECORDS_PER_REQUEST = 10
+    DEFAULT_API_BASE_URL = "https://api.airtable.com/"
 
     session: Session
+    api_base_url = DEFAULT_API_BASE_URL
+    api_url: str
     tiemout: TimeoutTuple
 
     def __init__(
         self,
         api_key: str,
         timeout: Optional[TimeoutTuple] = None,
         retry_strategy: Optional[Retry] = None,
+        api_base_url: Optional[str] = None,
```

Comment:
I actually kinda like the name `endpoint_url` (borrowing from the official Airtable library) since "Api" and "Base" are both overloaded terms in this context. Would you object to renaming it?

Reply:
No objections, what do you think about the naming of `api_url`?

Reply:
https://github.com/gtalarico/pyairtable/pull/243/commits/d1f732524461e27d19a350805bf415f1f116516d